tests/docs_test.go

```diff
@@ -0,0 +1,71 @@
+// +build system
+// +build docs
+
+package tests
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/mccanne/zq/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var doctests = []struct {
+	name string
+	cmd  string
+
+	expected string
+}{
+	{
+		"value-match",
+		"zq -f table '10.150.0.85' ../../zq-sample-data/zeek-default/*.log",
+		"value-match.table",
+	},
+	{
+		"quoted-word",
+		`zq -f table '"O=Internet Widgits"' ../../zq-sample-data/zeek-default/*.log`,
```

Comment:
I noticed the use of backticks here. I did some web searching and learned how this delivers an "uninterpreted string", hence validating my theory that this was a way to have both `'` and `"` within the string. Assuming I stick with doing this all in Go, any reason for me not to just use backticks for all these to be consistent?

Reply:
>  any reason for me not to just use backticks for all these to be consistent

No reason that I can see.